src/scripts/activate-user.ts

```diff
@@ -0,0 +1,71 @@
+import { Argv } from 'yargs';
+import { RootCommand } from '..';
+
+import { OktaUserService, User, UserService } from './services/user-service';
+import { oktaManageClient } from './services/client-service';
+import * as TE from 'fp-ts/lib/TaskEither';
+import * as E from 'fp-ts/lib/Either';
+import * as O from 'fp-ts/lib/Option';
+import { flow, pipe } from 'fp-ts/lib/function';
+import * as Console from 'fp-ts/lib/Console';
+
+export const activateUser = (
+  service: UserService,
+  userId: string
+): TE.TaskEither<string, User> =>
+  pipe(
+    userId,
+    service.getUser,
+    TE.chain(
+      flow(
+        O.fold(
+          () => TE.left(`User [${userId}] does not exist. Cannot activate.`),
+          // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+          () => service.activateUser(userId)
+        )
+      )
+    ),
+    TE.chainFirstIOK((user) =>
+      Console.info(`Activated [${user.id}] [${user.email}].`)
+    )
+  );
+
+export default (
+  // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+  rootCommand: RootCommand
+): Argv<{
+  readonly clientId: string;
+  readonly privateKey: string;
+  readonly organisationUrl: string;
+  readonly userId: string;
+}> =>
+  rootCommand.command(
+    'activate-user [user-id]',
+    'activates the specified user, only works if user currently has the status: staged or deprovisioned.',
```

Comment:
**question:** do you think that we should check the status up in `activateUser` so we can provide a better error message?

Reply:
actually yes i do. because okta doesn't give a specific error message as to why it failed

Reply:
I'll make this change

Reply:
added in commit: e3747a4725da06fa2ac00eb63f1015edda638086